inspect.go

```diff
@@ -0,0 +1,77 @@
+//
+// MinIO Object Storage (c) 2021 MinIO, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package madmin
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+)
+
+// InspectOptions provides options to Inspect.
+type InspectOptions struct {
+	Volume, File string
+}
+
+// Inspect makes an admin call to download a raw files from disk.
+func (adm *AdminClient) Inspect(ctx context.Context, d InspectOptions) (key [32]byte, c io.ReadCloser, err error) {
+	path := fmt.Sprintf(adminAPIPrefix + "/inspect-data")
+	resp, err := adm.executeMethod(ctx,
+		http.MethodGet, requestData{
+			relPath: path,
+			queryValues: url.Values{
```

Comment:
Let's use golang API to work with url.Values,

```
q := make(url.Values)
q.Set("volume", d.Volume)
q.Set("file", d.File)
```

Reply:
Not sure I understand the difference, but no problem.

Reply:
```
// Set sets the key to value. It replaces any existing
// values.
func (v Values) Set(key, value string) {
	v[key] = []string{value}
}
```
So yeah, the same AFAICT.

Reply:
> So yeah, the same AFAICT.

yeah, it looks like the same :) I thought maybe Golang is doing something extra